torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -394,9 +394,20 @@ def my_script_func(tensor):
 
 
 expected_err = "Expected error"
+
+# Note that it needs to inherit from Exception, not BaseException. See comment
+# in rpc/internal.py
+class CustomException(Exception):
+    def __init__(self, bool, msg):
+        self.bool = bool
+        super().__init__(msg)
+
 def raise_func():
     raise ValueError(expected_err)
 
+def custom_raise_func():
+    raise CustomException(True, "foo")
```

Comment:
N00b question, looks like we don't check this exception below. What's the purpose of creating a new customized exception?

Reply:
it is used in the string check - self.assertTrue("CustomException" in msg)

Reply:
Opps, my bad.